internal/runnerinstall/nomad.go

```diff
@@ -43,6 +43,7 @@ type NomadConfig struct {
 	CsiExternalId        string            `hcl:"nomad_csi_external_id,optional"`
 	CsiPluginId          string            `hcl:"nomad_csi_plugin_id,required"`
 	CsiSecrets           map[string]string `hcl:"nomad_csi_secrets,optional"`
+	CsiVolumeName        string            `hcl:"nomad_csi_volume_name,optional"`
```

Comment:
nit, let's group this with the other `csi_volume_` options. & maybe just `CsiVolume` without the `Name`? Since `HostVolume` isn't `HostVolumeName`

Reply:
Resolved in https://github.com/hashicorp/waypoint/pull/3546/commits/82a595eb404e8bd86ba9d7b6276ba4e894ac9137